__tests__/testUtils.js

```diff
@@ -47,7 +47,7 @@ export function createInnerSlider({ noOfSlides, ...settings }) {
     createReactSliderChildren(noOfSlides)
   );
   return (
-    <InnerSlider {...settings} ref={slider => (this.innerSlider = slider)}>
+    <InnerSlider {...settings} ref={React.createRef()}>
```

Comment:
@maddhruv This might break for older version of react.

Can you update the fix to make it work for both older and newer versions of react